contracts/cairo/deploy.sh

```diff
@@ -6,24 +6,18 @@ PURPLE='\033[1;34m'
 PINK='\033[1;35m'
 COLOR_RESET='\033[0m'
 
-HERODOTUS_FACTS_REGISTRY=0x01b2111317EB693c3EE46633edd45A4876db14A3a53ACDBf4E5166976d8e869d
-MM_ETHEREUM_WALLET=0xE8504996d2e25735FA88B3A0a03B4ceD2d3086CC
-NATIVE_TOKEN_ETH_STARKNET=0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7
+if [ -f ./contracts/cairo/.env ]; then
+    echo "Sourcing .env file..."
+    source ./contracts/cairo/.env
+else
+    echo "Error: .env file not found!"
+    exit 1
+fi
 
 cd "$(dirname "$0")"
 
-load_env() {
-    unamestr=$(uname)
-    if [ "$unamestr" = 'Linux' ]; then
-      export $(grep -v '^#' ../../mm-bot/.env | xargs -d '\n')
-    elif [ "$unamestr" = 'FreeBSD' ] || [ "$unamestr" = 'Darwin' ]; then
-      export $(grep -v '^#' ../../mm-bot/.env | xargs -0)
-    fi
-}
-load_env
-
```

Comment:
I would recommend merging PR #63 before trying to merge this change

Reply:
I'm going to revert this; actually, I tried to preemptively address conflicts by copying what you did in the other pull request.